src/pages/gallery/gallery.tsx

```diff
@@ -29,12 +29,12 @@ export default function Gallery({ games, tags }: { games: GameMetadata[], tags:
 
 	function filterTags(games: GameMetadata[]): void {
 		if (tagFilter === "") {
-			let otherGames = [...games];
+			let otherGames = [...games].map(game => ({ ...game, show: true }));
```

Comment:
Can this be expressed without the elipsis syntax?  I do not this the conciseness here outweighs readability.